src/main/java/nl/tudelft/cse1110/andy/execution/step/SourceCodeSecurityCheckStep.java

```diff
@@ -64,7 +66,14 @@ private boolean checkForKeywords(String code, ResultBuilder result) {
                 "getModifiers", reflectionMsg,
                 "reflect", reflectionMsg,
                 "setAccessible", reflectionMsg
-        );
+        ));
+        keywords.put("ChromeDriver", seleniumUnsupportedDriverMsg);
+        keywords.put("ChromiumDriver", seleniumUnsupportedDriverMsg);
+        keywords.put("EdgeDriver", seleniumUnsupportedDriverMsg);
+        keywords.put("FirefoxDriver", seleniumUnsupportedDriverMsg);
+        keywords.put("InternetExplorerDriver", seleniumUnsupportedDriverMsg);
+        keywords.put("OperaDriver", seleniumUnsupportedDriverMsg);
+        keywords.put("SafariDriver", seleniumUnsupportedDriverMsg);
```

Comment:
Why didn't you put those new keywords inside `Map.of`?

Reply:
Line 59:
```java
// An immutable map only supports 10 hardcoded KVs, we need more, hence HashMap wrapper.
```
The constructor only goes up to 10 keys and values and with these we have more.

Reply:
We should change this so that it has the same format as the current keyword list. (`Map.ofEntries`)